tests/kafkatest/version.py

```diff
@@ -91,5 +91,11 @@ def get_version(node=None):
 
 # 0.11.0.0 versions
 V_0_11_0_0 = KafkaVersion("0.11.0.0")
-LATEST_0_11_0 = V_0_11_0_0
+V_0_11_0_1 = KafkaVersion("0.11.0.1")
+V_0_11_0_2 = KafkaVersion("0.11.0.2")
+LATEST_0_11_0 = V_0_11_0_2
 LATEST_0_11 = LATEST_0_11_0
+
+# 1.0.0 versions
+V_1_0_0 = KafkaVersion("1.0.0")
+LATEST_1_0 = V_1_0_0
```

Comment:
Maybe just add `LATEST_1_1` here as well, though we are not going to use it yet?

Reply:
Ack

Reply:
That causes some problems with the dockerfile, since we'd have to download our test jar from the mirror, but it's not there yet. In fact, 1.0.0 is the latest package in the mirror.

Reply:
Those files are uploaded manually for the docker setup. I'll take care of it.